src/workerd/api/node/tests/zlib-nodejs-test.js

```diff
@@ -213,3 +213,10 @@ export const crc32Test = {
     });
   }
 }
+
+export const constantsTest = {
+  test() {
+    strictEqual(constants.Z_NEED_DICT, 2);
+    strictEqual(constants.BROTLI_DECODER_ERROR_FORMAT_EXUBERANT_META_NIBBLE, -3);
```

Comment:
We need better testing here. Particularly, snapshot testing in here would be the safest.

Reply:
Do you have an example of some existing snapshot testing in workerd? I'll snapshot all the constants then?

Reply:
Something like this should work deepStrictEqual(JSON.stringify(constants), "")